susemanager-utils/testing/docker/scripts/schema_idempotency_test_pgsql.py

```diff
@@ -0,0 +1,225 @@
+#!/usr/bin/python
+import argparse
+import difflib
+import distutils.version as DV
+import json
+import os
+import re
+import shutil
+import subprocess
+import sys
+
+
+def file_exists(path):
```

Comment:
Is there a particular reason not to use `os.path.isfile(path)`?

Reply:
To be honest, I don't know what I was thinking about x'D

Fixed and testing.